docs/faststore/docs/headless-cms/overview.mdx

```diff
@@ -61,12 +61,54 @@ Once editors click **Create Document** in the Headless CMS interface, they'll be
+
+The following table provides an overview of the standard Content Types, their IDs, scopes, and their respective purposes.
+
+> ℹ These standard Content Types are natively implemented. You can view their structure in the [content-types.json](https://github.com/vtex/faststore/blob/main/packages/core/cms/faststore/content-types.json) file.
+
+| **Content Type** | **ID** | **Scopes** | **Purpose** |
+| ------------------------------- | -------- | --------------- | --------------- |
+| Global Sections | `globalSections` | `global` | Defines reusable sections that appear across pages. |
+| Global Header Sections | `globalHeaderSections` | `global` | Defines the content for the global header, displayed at the top of every page. |
+| Global Footer Sections | `globalFooterSections` | `global` | Defines the content for the global footer, displayed at the bottom of every page. |
+| Landing Page | `landingPage` | `landing` and `custom` | Creates custom landing pages with SEO optimization. |
+| Home | `home` | `home` | Defines the structure and SEO metadata for the home page. |
+| Product Page | `pdp` | `pdp` | Defines the structure and template for Product Detail Pages (PDP). |
+| Product List Page | `plp` | `plp` | Defines the structure and settings for Product Listing Pages (PLP). |
+| Search Page | `search` | `plp` and `search` | Defines the structure and settings for search results pages. |
+| Login | `login` | - | Defines the structure for the login page. |
+| Error 500 | `500` | - | Defines the structure for the 500 Internal Server Error page. |
+| Error 400 | `400` | - | Defines the structure for the 400 pages, including the 404 Not Found Error page. |
+
 ### Sections
 
-For each Content Type, different **Sections** will be available to compose that page. Sections represent the content structure of a React component, for example, the [Alert](https://developers.vtex.com/docs/guides/faststore/molecules-alert).
+Each Content Type has different Sections available to build a page. Sections represent key React components, such as Hero or Product Gallery, manageable within the CMS. Editors can visually configure and arrange storefront Sections without needing to code. For example, the [Alert component](https://developers.vtex.com/docs/guides/faststore/molecules-alert) is a FastStore UI component and Section that can be used in various Content Types.
 
 ![CMS Section](https://vtexhelp.vtexassets.com/assets/docs/src/cms-sections___ccc8b00d73f222d249a009eb195c8e0c.png)
 
-Sections can be reused on different pages of an ecommerce. You are the one who will choose which Sections will be available at the CMS. Check the following example of the [Hero Section](https://developers.vtex.com/docs/guides/faststore/organisms-hero) being used:
+Sections are reusable across different pages. Developers determine which sections are available in the CMS. Below is an example of the [Hero Section](https://developers.vtex.com/docs/guides/faststore/organisms-hero):
```

Comment:
I don't know if we should link the component here, since we are referring to the Hero section specifically.